### ISSUE TITLE
Support for PHP 8.3

### ISSUE THREAD

**Issue by onlime** [poliander/cron#23]
Please update `composer.json` to support PHP 8.3

I have tested `poliander/cron` on latest PHP 8.3.0-dev (Homebrew) and all tests seem to run through fine:

```bash
$ php -v
PHP 8.3.0-dev (cli) (built: Oct 27 2023 15:45:44) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.3.0-dev, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.0-dev, Copyright (c), by Zend Technologies

$ ./vendor/bin/phpunit 
PHPUnit 9.6.13 by Sebastian Bergmann and contributors.

...............................................................  63 / 120 ( 52%)
.........................................................       120 / 120 (100%)

Time: 00:00.008, Memory: 6.00 MB

OK (120 tests, 224 assertions)
```

**Comment by poliander** [comment]
Preview is in 3.1 branch, includes removal of PHP 7.4/8.0 support and update to PHPUnit 10. Feedback is appreciated, but release of v3.1 will be as soon as PHP 8.3 is available (and GitHub Actions support)

**Comment by poliander** [comment]
I just released v3.1.0
